lib/platform-stats.js

```diff
@@ -15,15 +15,28 @@ export const updatePlatformStats = async (pgClient, honestMeasurements) => {
  * @param {import('./preprocess').Measurement[]} honestMeasurements
  */
 export const updateDailyStationStats = async (pgClient, honestMeasurements) => {
-  // TODO: when we add more fields, we will update the ON CONFLICT clause
-  // to update those fields, and we won't just use a Set for the stationIds
-  // which currently removes all granular measurement details
-  const stationIds = [...new Set(honestMeasurements.map(m => m.stationId))]
+  const stationIds = []
+  const counts = []
```

Comment:
can just use one data structure (see following suggestions)

Reply:
This would certainly be a more readable way, and what I started with originally, but it would be less efficient as we'd have to loop through the number of station IDs twice.

If we don't care about that since it's already a smaller, aggregated data structure, happy to make this change!

Reply:
To me optimization unless you've measured it and discovered a notable impact on performance

Reply:
If no-one notices your code being more efficient, then to me, it only has become less maintainable

Reply:
Fair point! Addressed in https://github.com/filecoin-station/spark-evaluate/pull/200/commits/afda88ed38927fcec00cc463f02538d86676f33e